roles/etcd/tasks/join_etcd-events_member.yml

```diff
@@ -24,7 +24,7 @@
   when: target_node == inventory_hostname
 
 - name: Join Member | Ensure member is in etcd-events cluster
-  shell: "{{ bin_dir }}/etcdctl --no-sync --endpoints={{ etcd_events_access_addresses }} member list | grep -q {{ etcd_events_access_address }}"
+  shell: "{{ bin_dir }}/etcdctl --no-sync --endpoints={{ etcd_events_access_addresses }} member list | grep -q {{ etcd_events_access_addresses }}"
```

Comment:
why is etcd_events_access_addresses? It include all ip list. so etcd_events_access_address shoud be define in roles/kubespray-defaults/defaults/main.yaml.  example etcd_events_access_address: {{ etcd_access_address}}

Reply:
yes, you are right, etcd_events_access_addresses should be defined in  roles/kubespray-defaults/defaults/main.yaml. But it isn't.

Reply:
etcd_events_access_address should in join_etcd-events_member.yml. But it also be defined in roles/kubespray-defaults/defaults/main.yaml.

Reply:
yes. but we need to define it in the same file.

Reply:
yes, we need to define etcd_events_access_address

Reply:
Should I undo this bug?

Reply:
no， you can modify the title.